server/src/models/OpenStax/config.ts

```diff
@@ -168,40 +232,37 @@ export default class Config {
             }
           },
         },
+        isSolutionPublic: (content) =>
```

Comment:
If all of its child solutions are public, the entire question set should be public too. This check helps avoid creating private content.json that only contains `{}` and makes question set a `SupportedLibrary`.